File: AzW.Model/Provision/Subnet.cs

namespace AzW.Model
 {
     public class Subnet
     {
         public string Name { get; set; }
         public string AddressSpace { get; set; }
         public string NSGName { get; set; }

         public string[] ServiceEndpointTargetServices { get; set; }
     }
 }

File: src/Portal/src/models/Subnet.js
import ResourceType from './ResourceType'
export default class Subnet
{
    constructor (){
        this.GraphModel = {
            Id: '',
            ResourceType: ResourceType.Subnet(),
            DisplayName: '',
            IsGatewaySubnet: false,
            VNetAddressSpace: '10.0.0.0/16',
            SubnetsAndCidrs: []
        };

        this.ProvisionContext = {
            ResourceType: ResourceType.Subnet(),

            Name: 'subnet-web',
            AddressSpace: '10.0.0.0/27',
            NSGName: '',
            ServiceEndpointTargetServices:[]
        };
        this.CalculatorContext = {
            IsFree: true,
            Tier: ''
        }
    }
}